### PR TITLE
Sos challenge fix

### PR DESCRIPTION
**Pull request by ablack3**
Just looking at the diff...

### CODE REVIEW

File: R/Incremental.R [hunk 1]
```diff
@@ -187,7 +187,7 @@ saveIncremental <- function(data, fileName, ...) {
     if (length(idx) > 0) {
       previousData <- previousData[-idx, ]
     }
-    data <- dplyr::bind_rows(previousData, data)
```
Comment (gowthamrao) [inline R/Incremental.R:190]: @ablack3  isnt the output of rbind not tibble? if yes, i think the loss of type stability may cause other problems

Comment (ablack3) [reply R/Incremental.R:190]: Yea I'm removing the strictness imposed by dplyr::bind_rows. I don't think my fix is actually a good idea :)
It needs more investigation to find a better fix. But what I did at least allowed the code to run for the SOS study.

**Comment by anthonysena** [comment]
@ablack3 sorry for the delay in review here. Let me know if this needs a review or if we should close it out.

**Comment by ablack3** [comment]
I think I was just viewing the diff. I'll close it.

**Pull request closed without merging**